### PR TITLE
Beacon cleanup

### PR DESCRIPTION
**Pull request by arnetheduck**

### CODE REVIEW

**Comment by tersec** [comment]
Seems to have some, but not all of, https://github.com/status-im/nim-beacon-chain/pull/115 ?

File: tests/simulation/start.sh [hunk 1]
```diff
@@ -37,13 +37,13 @@ if [[ -z "$SKIP_BUILDS" ]]; then
 fi
 
 if [ ! -f $STARTUP_FILE ]; then
-  $VALIDATOR_KEYGEN_BIN --validators=$NUMBER_OF_VALIDATORS --outputDir="$SIMULATION_DIR" # --startupDelay=2
+  $VALIDATOR_KEYGEN_BIN --validators=$NUMBER_OF_VALIDATORS --outputDir="$SIMULATION_DIR"
```
Comment (mratsim) [inline tests/simulation/start.sh]: I think it's good to keep it so that we don't need to track it down again. (Or it can be another env variable)

Comment (arnetheduck) [reply tests/simulation/start.sh]: it moved below and is called genesisoffset (because I did the code change first and didn't see this one)..

**Comment by arnetheduck** [comment]
oops, missed #115 when pushing this,rebased

**Pull request merged**